source/macos/AdaptiveCards/AdaptiveCards/Views/ACRButton.swift

```diff
@@ -111,87 +100,106 @@ class ACRButton: FlatButton, ImageHoldingView {
         textColor = activeTextColor
     }
     
-    private func setupButtonStyle(style: ActionStyle = .default) {
+    private func setupButtonStyle() {
         // Common styling b/w all Action Style
-        backgroundColor = .white
-        buttonColor = .white
-        activeTextColor = .white
-        
-        switch style {
+        switch buttonActionStyle {
         case .default:
+            // backgroundColor
+            backgroundColor = buttonConfig.defaultButtonConfig.backgroundColor
+            
             // borderColor
-            borderColor = ColorUtils.color(from: Constants.blueColorCode) ?? .clear
+            borderColor = buttonConfig.defaultButtonConfig.borderColor
             inActiveborderColor = borderColor
-            activeBorderColor = ColorUtils.color(from: Constants.darkBlueColorCode) ?? .clear
+            activeBorderColor = buttonConfig.defaultButtonConfig.activeBorderColor
```

Comment:
This function can be much simpler by having `switch` to decide the `colorConfig` and set the properties appropriately using the resolvedColorConfig